bokeh/embed/standalone.py

```diff
@@ -123,11 +123,8 @@ def components(models: Union[ModelLike, ModelLikeCollection], wrap_script: bool
     scripts tags. The widgets and tables resources are only necessary if the components
     make use of widgets and tables.
 
-    .. code-block:: html
-
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-x.y.z.min.js"></script>
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-x.y.z.min.js"></script>
-        <script src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-x.y.z.min.js"></script>
+    .. bokehjs-script-tag::
+        :js_files: bokeh-widgets,bokeh-tables
```

Comment:
This loads:
```html
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.2.min.js"
        integrity="sha384-ufR9RFnRs6lniiaFvtJziE0YeidtAgBRH6ux2oUItHw5WTvE1zuk9uzhUU/FJXDp"
        crossorigin="anonymous"></script>
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.2.min.js"
        integrity="sha384-8QM/PGWBT+IssZuRcDcjzwIh1mkOmJSoNMmyYDZbCfXJg3Ap1lEvdVgFuSAwhb/J"
        crossorigin="anonymous"></script>
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.2.min.js"
        integrity="sha384-Jm8cH3Rg0P6UeZhVY5cLy1WzKajUT9KImCY+76hEqrcJt59/d8GPvFHjCkYgnSIn"
        crossorigin="anonymous"></script>
```

And `.. bokehjs-script-tag::` with no `js_files` option would load:
```
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.2.min.js"
        integrity="sha384-ufR9RFnRs6lniiaFvtJziE0YeidtAgBRH6ux2oUItHw5WTvE1zuk9uzhUU/FJXDp"
        crossorigin="anonymous"></script>
...
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-api-2.0.2.min.js"
        integrity="sha384-Zat0VmMWmxj1LsoPt4eZNROXIBlbHawNk70+fMuyPdrkf4bxPfol5eSjkM+2+Aql"
        crossorigin="anonymous"></script>
```